stko/utilities/utilities.py

```diff
@@ -652,10 +652,10 @@ def get_long_bond_ids(mol, reorder=False):
 
 def calculate_dihedral(pt1, pt2, pt3, pt4):
     """
-    Calculate the dihedral between four points.
+    Calculate the dihedral between four points in degrees.
 
     Uses Praxeolitic formula --> 1 sqrt, 1 cross product
-    Output in range (-pi to pi).
+    Output in range (-180 to 180).
```

Comment:
This is not a comment for these changes, but we shuold clean up this code and remove the commented out code rather than straight copy/paste

Reply:
Agreed, but not going to do here.